lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_remove_testers_action.rb

```diff
@@ -26,9 +26,13 @@ def self.run(params)
           UI.user_error!("A maximum of 1000 testers can be removed at a time.")
         end
 
-        UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
-
-        count = fad_api_client.remove_testers(params[:project_number], emails)
+        if blank?(params[:group_alias])
+          UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
+          count = fad_api_client.remove_testers(params[:project_number], emails)
+        else
+          UI.message("⏳ Removing testers from group #{params[:group_alias]}...")
+          fad_api_client.remove_testers_from_group(params[:project_number], params[:group_alias], emails)
+        end
 
         UI.success("✅ #{count} tester(s) removed successfully.")
```

Comment:
`count` won't be defined here when testers are removed from group. Actually, will `count` even be defined when removing testers from the project since it's declared in the scope of the if/else? I guess Ruby may allow you to get away with that 🤔

Reply:
Fixed by removing the message inside the `if`/`else`